core/server/worker/src/test/java/alluxio/worker/block/RegisterStreamerTest.java

```diff
@@ -77,16 +76,9 @@ public class RegisterStreamerTest {
   private GrpcChannel mChannel;
 
   @Test(timeout = 5000)
-  public void testRegisterTimeOutConcurrentRequest() throws Exception {
+  public void testRegisterTimeOutConcurrentRequests() throws Exception {
```

Comment:
nit: `registerTimeoutConcurrentRequests`

Reply:
and this applies to all other test methods

Reply:
Fixed in new commit